lighthouse-core/config/default-config.js

```diff
@@ -146,6 +146,7 @@ const defaultConfig = {
       'meta-elements',
       'script-elements',
       'iframe-elements',
+      'forms',
```

Comment:
maybe there was previous discussion I missed, but my immediate reaction is that this should be `form-elements`, for consistency. 

even if it's not strictly `<form>` nodes, it is the collection of DOM elements commonly found in forms in an audit-agnostic structure similar to all our other `*-elements` gatherers.

any link I can follow or light to shed on the use of just `forms`? :)

Reply:
@patrickhulce  originally it was form elements, and we changed the way we collected them and I got a suggestion to change the name to forms. but what you are saying makes sense so let me check in with @connorjclark to get his opinion on this

reference: https://github.com/GoogleChrome/lighthouse/pull/11062#discussion_r452499000

Reply:
ah, yeah @connorjclark just `forms` feels a bit out of balance to me. was there something specific you had in mind for dropping `-elements`? :)

I mean they *are* still form elements, we just also capture some parent-child relationships along with them. (in addition to being a generic name for the elements that live inside forms 😆 )

Reply:
@patrickhulce you're very convincing.

sorry George, but can we go back to `form-elements`?

Reply:
sounds good lol @connorjclark